### PR TITLE
Klarocca/vqa whitelabel

### PR DESCRIPTION
**Pull request by kimlarocca**

### CODE REVIEW

File: app/styles/themes/white-label/_theme.overrides.scss [hunk 1]
```diff
@@ -56,26 +70,255 @@
...
+.o-rte-text > ol li,
+.o-rte-text > ul li,
+.o-rte-text > p {
+  @include typeface(body, 5);
+}
+
+.o-rte-text > h1, .o-rte-text > h2, .o-rte-text > h3, .o-rte-text > h4, .o-rte-text > h5, .o-rte-text > h6 {
+  padding-top: var(--space-4);
+}
+.c-block__dek p,
```
Comment (bkammin) [inline app/styles/themes/white-label/_theme.overrides.scss:162]: add a blank/spacing line above this line

File: app/styles/themes/white-label/_theme.overrides.scss [hunk 2]
```diff
@@ -56,26 +70,255 @@
...
+
+.o-rte-text > dl dd,
+.o-rte-text > dl dt,
+.o-rte-text > ol li,
+.o-rte-text > ul li,
+.o-rte-text > p {
+  @include typeface(body, 5);
+}
+
+.o-rte-text > h1, .o-rte-text > h2, .o-rte-text > h3, .o-rte-text > h4, .o-rte-text > h5, .o-rte-text > h6 {
```
Comment (bkammin) [inline app/styles/themes/white-label/_theme.overrides.scss:159]: separate class/tag specifiers onto the their own lines like you did just above here

File: app/styles/themes/white-label/_theme.overrides.scss (hunk 1 above)
Comment (kimlarocca) [reply app/styles/themes/white-label/_theme.overrides.scss:162]: ok!

File: app/styles/themes/white-label/_theme.overrides.scss (hunk 2 above)
Comment (kimlarocca) [reply app/styles/themes/white-label/_theme.overrides.scss:159]: good catch, I copied and pasted this from the browser!

**Comment by kimlarocca** [comment]
@bkammin made your formatting changes plus some more updates

File: app/styles/themes/white-label/_theme.overrides.scss [hunk 3]
```diff
@@ -281,12 +296,14 @@
...
-  border: 2px solid rgb(var(--color-white));
+  border: 2px solid;
+  border-color: rgb(var(--color-white));
 }
 
 .c-newsletter-form__terms input[type="checkbox"]:checked {
   background-color: rgb(var(--color-ceramic-blue));
-  border: 2px solid rgb(var(--color-white));
+  border: 2px solid;
+  border-color: rgb(var(--color-white));
```
Comment (bkammin) [inline app/styles/themes/white-label/_theme.overrides.scss:306]: I think I had to make this same fix a while back for the same reason -- worked locally, but not on demo. It looked to me like the color was getting dropped when the CSS was compiled and compressed, not sure why.

**Pull request merged**